test/Directory.Build.props

```diff
@@ -43,6 +43,8 @@
            but they need to be migrated. Excluding these classes should have very minimal effect on code coverage.
       -->
       <Exclude>[*]Microsoft.ML.*Contracts*,[*]Microsoft.ML.Internal.Utilities*,[*]Microsoft.ML.Data.VBuffer*</Exclude>
+      <ExcludeByAttribute>Obsolete,ExcludeFromCodeCoverage</ExcludeByAttribute>
+      <ExcludeByFile>$(BaseOutputPath)..\src\Microsoft.ML.Onnx\OnnxMl.cs,$(BaseOutputPath)..\src\Microsoft.ML.TensorFlow\TensorFlow\Buffer.cs,$(BaseOutputPath)..\src\Microsoft.ML.TensorFlow\TensorFlow\Tensor.cs,$(BaseOutputPath)..\src\Microsoft.ML.TensorFlow\TensorFlow\TensorGeneric.cs,$(BaseOutputPath)..\src\Microsoft.ML.TensorFlow\TensorFlow\Tensorflow.cs</ExcludeByFile>
```

Comment:
>\src\Microsoft.ML.TensorFlow\TensorFlow\TensorGeneric.cs [](start = 230, length = 56)

Can we include this file? as I see that we have a test that goes through code in this file. 
[https://github.com/dotnet/machinelearning/blob/8c0a111a596a2e32847ae86c67bb892edcd7647e/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L105](https://github.com/dotnet/machinelearning/blob/8c0a111a596a2e32847ae86c67bb892edcd7647e/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L105)

It would be nice to see the code coverage for this file too.

Reply:
Done.